conf/supervisord-dev.conf

```diff
@@ -0,0 +1,27 @@
+[supervisord]
+nodaemon=true
+environment=PYTHONUNBUFFERED="1"
+logfile=/dev/null
+logfile_maxbytes=0
+
+[program:web]
+command=newrelic-admin run-program gunicorn --paste conf/development.ini
```

Comment:
This is really the only difference between prod and dev. Maybe this could be specified via an environment variable?